main.tf

```diff
@@ -1,24 +1,36 @@
 /**
- * Creates and configures an S3 bucket for storing logs from various AWS
- * services and enables CloudTrail on all regions. Logs will expire after a
- * default of 90 days. Includes support for sending CloudTrail events to a
- * CloudWatch Logs group.
+ * Supports two main uses cases:
  *
- * Logging from the following services is supported:
+ * 1. Creates and configures a single private S3 bucket for storing logs from various AWS services, which are nested as bucket prefixes, and enables CloudTrail on all regions. Logs will expire after a default of 90 days, with option to configure retention value. Includes support for sending CloudTrail events to a CloudWatch Logs group.
+ *
+ * 2. Creates and configures a single private S3 bucket for a single AWS service, and enables CloudTrail on all regions. Logs will expire after a default of 90 days, with option to configure retention value. Includes support for sending CloudTrail events to a CloudWatch Logs group.
+ *
+ * Logging from the following services is supported for both cases:
  *
  * * [CloudTrail](https://aws.amazon.com/cloudtrail/)
  * * [Config](https://aws.amazon.com/config/)
- * * [Elastic Load Balancing](https://aws.amazon.com/elasticloadbalancing/)
+ * * [Elastic Load Balancing (and Application Load Balancing (ALB))](https://aws.amazon.com/elasticloadbalancing/)
  * * [RedShift](https://aws.amazon.com/redshift/)
  * * [S3](https://aws.amazon.com/s3/)
  *
- * ## Usage
+ * ## Usage for a single log bucket to store logs from all services as bucket prefixes
  *
+ *     # Turns on cloudtrail by default, and allows all services to log to bucket
  *     module "aws_logs" {
  *       source                  = "trussworks/logs/aws"
  *       s3_bucket_name          = "my-company-aws-logs"
  *       region                  = "us-west-2"
- *       s3_log_bucket_retention = 90
+ *     }
+ *
+ * ## Usage for a single log bucket to store logs from a *single* service
+ *
+ *     # Turns on cloudtrail by default, and allows only the service specified (elb in this case) to log to the bucket
+ *     module "aws_logs" {
+ *       source                  = "trussworks/logs/aws"
+ *       s3_bucket_name          = "my-company-aws-logs-elb"
+ *       region                  = "us-west-2"
+ *       enable_all_services     = false
+ *       enable_elb              = true
```

Comment:
These equals signs should be aligned one space from the longest key: `enable_all_services`.

Reply:
fixed